types/log4js.d.ts

```diff
@@ -191,6 +191,10 @@ export interface FileAppender {
   keepFileExt?: boolean;
   // (defaults to .) the filename separator when rolling. e.g.: abc.log`.`1 or abc`.`1.log (keepFileExt)
   fileNameSep?: string;
+  /**
+   * (defaults to false) remove embedded ANSI color sequence
+   */
```

Comment:
To retain the consistent commenting style in this file.
```suggestion
  // (defaults to false) remove embedded ANSI color sequence
```

Reply:
@lamweili Thank you for the feedback! I've just fixed it.